gradle-baseline-java/src/main/java/com/palantir/baseline/extensions/BaselineErrorProneExtension.java

```diff
@@ -97,14 +94,19 @@ public class BaselineErrorProneExtension {
             "UnnecessaryParentheses",
             "ZoneIdOfZ");
 
-    private final ListProperty<String> patchChecks;
+    private final SuppressibleErrorProneExtension suppressibleErrorProneExtension;
 
-    public BaselineErrorProneExtension(Project project) {
-        patchChecks = project.getObjects().listProperty(String.class);
-        patchChecks.set(DEFAULT_PATCH_CHECKS);
+    /**
+     * .
+     * @deprecated Interact with SuppressibleErrorProneExtension instead. This only provided for backcompat.
+     */
+    @Deprecated
+    public final SetProperty<String> getPatchChecks() {
+        return suppressibleErrorProneExtension.getPatchChecks();
     }
 
-    public final ListProperty<String> getPatchChecks() {
-        return patchChecks;
+    public BaselineErrorProneExtension(SuppressibleErrorProneExtension suppressibleErrorProneExtension) {
+        this.suppressibleErrorProneExtension = suppressibleErrorProneExtension;
+        suppressibleErrorProneExtension.getPatchChecks().addAll(DEFAULT_PATCH_CHECKS);
```

Comment:
perhaps it already exists, but should we be worried that it's easy to wipe out the defaults just by doing `patchChecks = [ ... ] ` ?

Reply:
I think this is a valid thing to do, although probably not the right thing to do. I think we should just not advertise this ability at all.